### ISSUE TITLE
ucs_vlans and ucs_vnic_template not using -u username -k password arguments

### ISSUE THREAD

**Issue by minimalz** [CiscoUcs/ucsm-ansible#100]
We do not want to define the username and password within the playbook or a yml var file as we do not use role accounts in our environment, each admin uses their own credentials.  CLI arguments -u username and -k password do not appear to be used by the module.  We see the following errors based on how we define username and password in the playbook.  

CLI: ansible-playbook ucs_add_vlans_vnic.yml -vvvv -u username -k password

1) ucs_add_vlans_vnic.yml
username: "{{ ansible_ssh_user }}"
password: "{{ ansible_ssh_password }}"

The task includes an option with an undefined variable. The error was: 'ansible_ssh_password' is undefined

2) ucs_add_vlans_vnic.yml
username:
password:

msg: '[ErrorCode]: 551[ErrorDescription]: Authentication failed'

3) ucs_add_vlans_vnic.yml
Username and password not defined in playbook.

msg: 'missing required arguments: password'

**Comment by dsoper2** [comment]
All the ansible_* variables are available to the playbooks.  A few things to note:
- ansible_ssh_pass is the variable (not password).  The full set of available variables is defined at https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html
- The -k option will prompt for a password (and normally you would not directly provide the password on the command line.  Here's example use with Ansible prompting for a password that I verified works with the modules:
ansible-playbook example.yml -k
SSH password: 
- Because the UCS modules would normally operate with connection=local, I typically use other custom defined variables since ansible_user or other ansible defined variables may behave in different ways with local vs. remote connections.  Here's an additional example command line and login information that I verified works with the modules and Ansible 2.7:
ansible-playbook example.yml -k -e ucs_username=admin
SSH password: 
  - using these login variables:
      hostname: "{{ ansible_host }}"
      username: "{{ ucs_username }}"
      password: "{{ ansible_ssh_pass }}"
